src/ring_theory/polynomial/pochhammer.lean

```diff
@@ -92,6 +92,31 @@ begin
     refl, },
 end
 
+lemma pochhammer_add_pochhammer_aux {R : Type*} [comm_semiring R] : ∀ (n : ℕ),
+  pochhammer R (n + 1) + (n + 1) * (pochhammer R n).comp (X + 1) =
+    (pochhammer R (n + 1)).comp (X + 1)
```

Comment:
I think this would be better phrased backwards, as:
```suggestion
lemma pochhammer_succ_comp_X_add_one {R : Type*} [comm_semiring R] : ∀ (n : ℕ),
  (pochhammer R (n + 1)).comp (X + 1) =
    pochhammer R (n + 1) + (n + 1) * (pochhammer R n).comp (X + 1) =
```

Reply:
It might also make sense to use `smul` instead of `(n + 1) *`